errors/no-single-file-deployments.md

```diff
@@ -0,0 +1,9 @@
+# No Single File Deployments
+
+#### Why This Error Occurred
+
+You attempted to create a Vercel deployment where the input is a file, rather than a directory. Previously this was allowed, however this behavior has been removed as of Vercel CLI vX.X.X because it exposed a potential security risk if the user accidentally created a deployment from a sensitive file.
```

Comment:
Let's remember to add the version here. Probably semver major?

Reply:
Tentatively set it to 24.0.0 since that will be the next major release.